app/views/players/show.html.haml

```diff
@@ -61,6 +61,9 @@
         - if @player.alt_hof == 'nl'
           Major League
         player, but he is in the Hall of Fame as a #{@player.alt_hof_name}.
+    - if @bbwaa_2017_returning
```

Comment:
@semanticart @mikowitz I'm using the `bbwaa_2017_returning` player scope here so I can show on a player page if he's returning to the ballot in 2017. This is always returning `true`, but basically what I want to ask is "is the player that I'm looking at included in the `bbwaa_2017_returning` group?" Is that relatively straight-forward?